src/Protocol/Http/Authenticator/JwtAuthenticator.php

```diff
@@ -30,24 +30,61 @@ class JwtAuthenticator implements AuthenticatorInterface
      */
     private $jwt;
 
+    /**
+     * @var string
+     */
+    private $jws = '';
```

Comment:
Please do not use the defaults values in property declaration.

Problem: we replace these parameters in the constructor.

Note: for `$jws`, we can use the null as default, and in next code easy check by empty parameter `!$this->jws`.

Example:

```php
class Some
{
    private $jws;
    private $jwsLifetime;
    private $jwsValidTo;
}
```

Thanks for understanding.